sources/dynamodb/README.md

```diff
@@ -32,16 +32,22 @@ To use the tool on a DynamoDB database (it will migrate all tables),
 run
 
 ```sh
-harbourbridge -driver=dynamodb
+harbourbridge schema -source=dynamodb
 ```
 
-Due the schemaless nature of DynamoDB, the tool infers the schema based on a
-certain amount of sampled data, by default, 100000 rows. If a table has more
-rows than the default value, we only use 100000 rows for estimating the schema.
-You can change this value via the flag `schema-sample-size`. For example,
+### Source profile (`-source-profile`)
+
+HarbourBridge accepts the following options for --source-profile for DynamoDB,
+specified as "key1=value1,key2=value,..." pairs:
```

Comment:
Maybe drop "specified as "key1=value1,key2=value,..." pairs"
since there is only one option to specify?

Reply:
Done.